airbyte-integrations/bases/source-acceptance-test/README.md

```diff
@@ -9,19 +9,27 @@ Test-specific documentation can be found [here](https://docs.airbyte.com/connect
 3. Build the connector docker image ( e.g.: `docker build . -t airbyte/source-pokeapi:dev`)
 4. Use one of the following ways to run tests (**from your connector project directory**)
 
-**Using python**
+### Using python
+_Note: these will assume that docker image for connector is already built_
+
+**Running the whole suite**
 ```bash
 python -m pytest integration_tests -p integration_tests.acceptance
 ```
-_Note: this will assume that docker image for connector is already built_
 
-**Using Gradle**
+**Running a specific test**
+```bash
+python -m pytest integration_tests -p integration_tests.acceptance -k "<TEST_NAME>"
```

Comment:
`-k`, of course :D

Reply:
You can use it to select multiple tests too! Alex showed me and I was amazed